### PR TITLE
Becatrue/118/change bus to channel

### LINKED ISSUE #118: Change the word "bus" -> "channel"
How about changing the term "bus" to "channel"?
Since now there is no concept of actual buses, so it sounds a little awkward.
_Broadcasting_ to a _channel_ would be natural and intuitive.

### PR DESCRIPTION
**Pull request by BECATRUE**
This PR will close #118.

I changed all `bus`es to `channel`s, and set channel names as `dbch` and logch`.

### CODE REVIEW

File: swift/app.py [hunk 1]
```diff
@@ -13,10 +13,10 @@ class BaseApp(QObject):
     """Base App class that all apps should inherit.
 
     Signals: 
-        broadcastRequested(str, str): A signal for broadcasting to a bus 
-          which contains the destination bus name and the message.
-        received(str, str): A signal for receiving a global signal from a bus
-          which contains the departure bus name and the message.
+        broadcastRequested(str, str): A signal for broadcasting to a channel 
+          which contains the destination channel name and the message.
```
Comment (kangz12345) [inline swift/app.py:17]: Now the expression "destination" becomes awkward. How about "the target channel"?

File: swift/app.py [hunk 2]
```diff
@@ -13,10 +13,10 @@ class BaseApp(QObject):
...
+        received(str, str): A signal for receiving a global signal from a channel
+          which contains the departure channel name and the message.
```
Comment (kangz12345) [inline swift/app.py:19]: How about "the source channel"?

File: swift/swift.py [hunk 3]
```diff
@@ -41,7 +41,7 @@ class AppInfo:
         pos: Position on the main window; refer to Qt.DockWidgetArea enum.
           Should be one of "left", "right", "top", or "bottom", case-sensitive.
           Otherwise, defaults to Qt.LeftDockWidgetArea.
-        bus: Buses which the app subscribes to.
+        channel: Channeles which the app subscribes to.
```
Comment (kangz12345) [inline swift/swift.py:44]: May be you have simply replaced the words 😄
Please change it to "Channels".

**Review by kangz12345 — CHANGES_REQUESTED** [review]

File: swift/swift.py (hunk 3 above)
Comment (BECATRUE) [reply swift/swift.py:44]: Ah.. I checked the other files, but I don't think I can do it here..😂

Comment (kangz12345) [reply swift/swift.py:44]: I think the other "channels" are fine!
Channeles -> Channels

**Comment by BECATRUE** [comment]
I updated it. @kangz12345

**Review by kangz12345 — APPROVED** [review]
LGTM

**Pull request merged**